apps/api/src/threads/threads.service.ts

```diff
@@ -1602,53 +1602,89 @@ export class ThreadsService {
         stream,
         initialMessage,
       )) {
-        chunkCount++;
-        if (!ttfbEnded) {
-          ttfbSpan.end();
-          ttfbEnded = true;
-        }
+        const chunkProcessingSpan = Sentry.startInactiveSpan({
```

Comment:
I think this can be an active span, no? Not totally sure how all the async stuff will roll up but it would be nice to have each chunk contain its children..